README.md

```diff
@@ -42,96 +76,62 @@ Returns an event
 
 #### Path parameters
 
-| Parameter | Type    | Required | Default | Description                        |
-| :-------- | :------ | :------- | :------ | :--------------------------------- |
-| eventId   | integer | yes      | none    | id of the event you are retrieving |
+| Parameter | Type   | Required | Default | Description                        |
+| :-------- | :----- | :------- | :------ | :--------------------------------- |
+| eventId   | string | yes      | none    | id of the event you are retrieving |
 
 #### Sample response
 
 ```json
 {
-  "eventId": 12,
-  "coupleName1": "Ted Baker",
-  "coupleName2": "Yves St-Laurent",
-  "email": "couple@email.com",
-  "date": "2022-08-12T20:17:46.384Z",
-  "venue": "Hack Reactor Auditorium",
-  "addressLine1": "119 Nueces St",
-  "addressLine2": "Room 1104",
+  "_id": "61b79b9e0ac02dbe3e12fd1b",
+  "coupleName1": "Ford",
+  "coupleName2": "Ferrari",
+  "email": "ford_ferrari_wedding@domain.com",
+  "date": "2022-12-13T00:00:00.000Z",
+  "venue": "The Big Venue",
+  "addressLine1": "123 Big Venue St.",
+  "addressLine2": "Suite 12",
   "city": "Austin",
   "state": "TX",
-  "zipCode": "78701",
-  "daysToEvent": 15,
+  "zip": "12345",
   "guestLimit": 100,
-  "rsvpDeadline": "2021-12-01T20:17:46.384Z",
-  "daysToRSVPDeadline": 12,
-  "inviteMessage": "Please join us for our special day",
-  "dashboardPhoto": "shorturl.at/oJW34",
-  "bannerPhoto": "shorturl.at/oJW34",
+  "rsvpDeadline": "2022-06-13T00:00:00.000Z",
+  "inviteMessage": "Please join us in celebrating our new life together.",
+  "dashboardPhotoURL": "shorturl.at/tuHZ3",
+  "bannerPhotoURL": "shorturl.at/tuHZ3",
   "galleryPhotos": {
-    "1": "shorturl.at/oJW34",
-    "2": "shorturl.at/oJW34",
-    "3": "shorturl.at/oJW34"
+    "1": "shorturl.at/tuHZ3",
+    "2": "shorturl.at/tuHZ3"
   },
   "colors": {
-    "1": "#F0DA32",
-    "2": "#87B3BF",
-    "3": "#6E4AFF",
-    "4": "#5337C9"
+    "1": "#FFFFFF",
+    "2": "#000000"
   },
+  "__v": 0,
   "guests": {
```

Comment:
Would we still want access to each guest's RSVP note? I noticed it was missing in the new readme

Reply:
Yes, it is included in the response if there is an `rsvpNote`. These are example guests that had not yet RSVP'd, and if a guest has not yet RSVP'd (rsvpStatus is 'pending'), there will be no `rsvpNote`.

If you go to the example for `GET /events/:eventId/guests` (line 246), you will see it in there.

Reply:
I've added another example to the md to make it more clear that guests that have RSVP'd will have their rsvpNote in there"